tests/UI/Http/Rest/Controller/JsonApiTestCase.php

```diff
@@ -32,15 +32,15 @@ protected function createUser(string $email = self::DEFAULT_EMAIL, string $passw
             $password
         );
 
-        /** @var CommandBus $commandBus */
-        $commandBus = $this->cli->getContainer()->get('tactician.commandbus.command');
+        /** @var MessageBusInterface $commandBus */
+        $commandBus = $this->cli->getContainer()->get('messenger.bus.command');
```

Comment:
Replace messenger bus in favor of the infrastructure one. This one shouldn't be public in the container. It fact it isn't and it's the reason why tests fails I think

Reply:
@jorge07 tests passed for now, but there is a problem with Github actions can You look at this?